api/services/gameService.js

```diff
@@ -73,8 +73,15 @@ module.exports = {
 
     if (game.players) {
       if (game.players.length < 2) {
-        throw new Error({ message: 'Cannot populate game without two players' });
-      }
+        /*no game was found*/
+        if(game.players.length == 0){
+          return { status: 400, message: 'Bad Request: Cannot find a game' };
```

Comment:
The reason that handling the issue in this specific way here is awkward is that this a helper method that isn't directly in controller code and doesn't have access to the `response` object, meaning whatever this returns doesn't inherently correspond to a specific response that ultimately gets sent to the client. So returning a differently shaped object here (rather than the Game that callers of this helper are expecting) will potentially goof the other places that use this method and it won't force them to return a `res.badRequest()`. That's why (per the comment below) we want to handle this change in the `spectate` endpoint file, rather than in this `gameService` helper.